src/i18n/strings/en_EN.json

```
@@ -3658,6 +3658,12 @@
         "toast_title": "Update %(brand)s",
         "unavailable": "Unavailable"
     },
+    "update_room_access_modal": {
+        "description": "To create a share link, you need to update the room access settings. This can impact the security level of the room. It is recommended to set it back to private if the link is not needed anymore.",
+        "dont_change_description": "Alternatively a different or new room can be used for the conference share link.",
```

```suggestion
        "dont_change_description": "Alternatively, you can hold this call in a different room.",
```

Reworded to use active voice